source/dem/dem.cc

```diff
@@ -1057,6 +1058,21 @@ DEMSolver<dim>::write_output_results()
       write_boundaries_vtu<dim>(
         data_out_faces, folder, time, iter, this->mpi_communicator);
     }
+  if (parameters.post_processing.force_chains)
+    {
+      // Force chains visualization
+      ParticlesForceChains<
+        dim,
+        Parameters::Lagrangian::ParticleParticleContactForceModel::hertz,
+        Parameters::Lagrangian::RollingResistanceMethod::no_resistance>
+        particles_force_chains_object(parameters);
+      particles_force_chains_object.calculate_force_chains(
+        contact_manager, simulation_control->get_time_step());
+
+      particles_force_chains_object.write_force_chains(this->mpi_communicator,
+                                                       folder,
+                                                       iter);
```

Comment:
Right now this is hardcoded to use only a single collision model. It is important to refactor this to enable all variants of the force model. Feel free to ask questions regarding this if you need help.